portfolio/src/main/webapp/script.js

```diff
@@ -59,7 +59,14 @@ function limitNumComments(){
 
 function createComment(titleText, bodyText, emailText,sentimentScore){
     const comment = document.createElement("div");
-    comment.className = "comment";
+    if(sentimentScore.toFixed(1) > .3){
+      comment.className = "posComment";
+    } else if (sentimentScore.toFixed(1) <=.3 && sentimentScore.toFixed(1) >= -.3) {
+      comment.className = "neuComment";
+    }
+    else{
```

Comment:
Move else to be on the line above (after the { ) and add a space in between else and the {